woocommerce/woocommerce-gutenberg-products-block
# Move Button and Label components to `@woocommerce/blocks-checkout` package

This PR is part of a series of smaller PRs that relocate certain components to the Checkout package. I will do 1-2 components per PR to ensure the diffs are manageable.

This PR contains the changes for the `Label` and `Button` components.



### How to test the changes in this Pull Request:

1. Smoke test of Blocks functionality.
2. Ensure critical flows work, such as adding items to cart from the All Products Block, continuing to Checkout from the Cart Block and placing an order in the Checkout Block.
2. Ensure labels are displaying correctly in blocks, compare to trunk and ensure nothing is different or missing.



### Changelog

> Move Button and Label components to `@woocommerce/blocks-checkout` package
